### ISSUE TITLE
Update to Slim 4?

### ISSUE THREAD

**Issue by samuelgfeller** [tuupola/slim-api-skeleton#54]
Thank you for this awesome skeleton-project. 

Have you planned to do an update to make it slim 4 compatible and with the (new) best practices ? 

I would be highly interested by it.

**Comment by tuupola** [comment]
Yes it is the plan. I already started the work but will finish it when I have new API project myself.

https://github.com/tuupola/slim-api-skeleton/tree/slim-4

**Comment by samuelgfeller** [comment]
Okay thank you I'm looking forward to it!
